src/main/java/com/acquia/http/HMACMessageCreator.java

```diff
@@ -102,15 +111,15 @@ public String createMessage(HttpServletRequest request) throws IOException {
     }
 
     /**
-     * Create the message based on the given HTTP request to be sent and the list of custom header names.
+     * Create response signature message
```

Comment:
No real reason other than that I have another method to create signable response message from httpResponse..
Custom headers are not quite relevant at this point, as it isn't being passed in as parameter anymore, but httpRequest is still relevant.. I guess I can put that in.